### PR TITLE
t2f tests should cover trips longer than 10 days

### PR DESCRIPTION
**Pull request by reinbach**
Concentrated on the calculations performed around itineraries.

A number of tests have TODO comments, where there is a question about the test results. Making me think I'm either making a mistake or have a lack of understanding.

### CODE REVIEW

File: EquiTrack/t2f/tests/test_dsa_calculations.py [hunk 1]
```diff
@@ -71,6 +164,940 @@ def setUp(self):
...
+            dsa_region=self.amsterdam
+        )
+        dsa_dto_list = [DSAdto(date(2017, 1, 1), itinerary)]
+        dsa = DSACalculator(self.travel)
+        self.assertEqual(
+            dsa.check_one_day_long_trip(dsa_dto_list),
+            dsa_dto_list
+        )
+
+    # TODO: Confirm that this is correct, Seems counterintuitive
```
Comment (reinbach) [inline EquiTrack/t2f/tests/test_dsa_calculations.py]: It looks like we're only checking the hours between 2 itineraries and not for all itineraries for the specific day. Is that correct?

File: EquiTrack/t2f/tests/test_dsa_calculations.py [hunk 2]
```diff
@@ -71,6 +164,940 @@ def setUp(self):
...
+            departure_date=datetime(2017, 1, 1, 12, 0, tzinfo=UTC),
+            dsa_region=self.budapest
+        )
+        dsa = DSACalculator(self.travel)
+        dto = DSAdto(date(2017, 1, 1), itinerary)
+        dto.dsa_amount = 0
+        dsa.add_same_day_travels(dto, 1)
+        self.assertEqual(dto.dsa_amount, 0)
+
+    # TODO: confirm that this is correct
```
Comment (reinbach) [inline EquiTrack/t2f/tests/test_dsa_calculations.py]: How come this is calculated for itineraies on different regions?

File: EquiTrack/t2f/tests/test_dsa_calculations.py [hunk 3]
```diff
@@ -71,6 +164,940 @@ def setUp(self):
...
+        self.assertEqual(dsa_dto_list[1].dsa_amount, 0)
+        for dto in dsa_dto_list[2:59]:
+            self.assertEqual(dto.dsa_amount, self.amsterdam.dsa_amount_usd)
+        for dto in dsa_dto_list[60:]:
+            self.assertEqual(
+                dto.dsa_amount,
+                self.amsterdam.dsa_amount_60plus_usd
+            )
+
+    # TODO: Confirm this is correct.
```
Comment (reinbach) [inline EquiTrack/t2f/tests/test_dsa_calculations.py]: This tests shows multiplier being added twice.

I think the logic can be tightened up a little, unless I'm missing something. But want to confirm the expected result first.

See https://github.com/unicef/etools/blob/0bfbfffa90edec41834a6a7da1d2f15aeab15c2e/EquiTrack/t2f/helpers/cost_summary_calculator.py#L332

File: EquiTrack/t2f/tests/test_cost_summary_calculator.py [hunk 4]
```diff
@@ -136,3 +213,202 @@ def test_cost_summary_calculator(self):
         calculator = CostSummaryCalculator(self.travel)
         # Should not raise TypeError
         calculator.get_cost_summary()
+
+    # TODO: confirm that this is correct
```
Comment (reinbach) [inline EquiTrack/t2f/tests/test_cost_summary_calculator.py]: Should a single itinerary record result in zero dsa total? I think my understanding is off, on how itinerary records are created/generated.

See https://github.com/unicef/etools/blob/7c04b805e54e5184ab40ea558a308b56ddc83558/EquiTrack/t2f/tests/test_dsa_calculations.py#L464

and https://github.com/unicef/etools/blob/7c04b805e54e5184ab40ea558a308b56ddc83558/EquiTrack/t2f/helpers/cost_summary_calculator.py#L226

File: EquiTrack/t2f/tests/test_dsa_calculations.py (hunk 1 above)
Comment (ntrncic) [reply EquiTrack/t2f/tests/test_dsa_calculations.py]: we can skip this. dsa calculations have been disabled

File: EquiTrack/t2f/tests/test_dsa_calculations.py (hunk 2 above)
Comment (ntrncic) [reply EquiTrack/t2f/tests/test_dsa_calculations.py]: skip this test too

File: EquiTrack/t2f/tests/test_dsa_calculations.py (hunk 3 above)
Comment (ntrncic) [reply EquiTrack/t2f/tests/test_dsa_calculations.py]: skip

File: EquiTrack/t2f/tests/test_cost_summary_calculator.py (hunk 4 above)
Comment (ntrncic) [reply EquiTrack/t2f/tests/test_cost_summary_calculator.py]: skip

**Pull request merged**